clients/python-wrapper/pyproject.toml

```
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["setuptools"]
```

For PEP 639 support.

```suggestion
requires = ["setuptools>=77"]
```